target/linux/ar71xx/image/generic-legacy-devices.mk

```diff
@@ -103,6 +103,12 @@ define LegacyDevice/AP143_16M
 endef
 LEGACY_DEVICES += AP143_16M
 
+define LegacyDevice/ANONABOX_PRO
+  DEVICE_TITLE := ANONABOX_PRO reference board (16MB flash)
```

Comment:
The page says that it has 64MB flash: https://www.anonabox.com/buy-anonabox-pro.html

And why is this a reference board? Looks to me like a normal consumer device